mmcv/parallel/distributed.py

```diff
@@ -18,6 +18,11 @@ class MMDistributedDataParallel(DistributedDataParallel):
     - It implement two APIs ``train_step()`` and ``val_step()``.
     """
 
+    def to_kwargs(self, inputs, kwargs, device_id):
+        #  Use `self.to_kwargs` instead of `self.scatter` in pytorch1.8
```

Comment:
Are there too many blanks between `#` and `Use`?

Reply:
sure